README.md

```diff
@@ -316,6 +316,71 @@ Library specific options:
     ZSTD_INCLUDE_DIR                - Directory that contains zstd.h
     ZSTD_LIBRARY                    - Path to libzstd.a/libzstd.so/ztd.lib
 
+### Compiling for Windows using MinGW on GNU/Linux
+
+It is highly recommeded to compile on Ubuntu 18.04 or newer.
```

Comment:
If you just install cmake from the repos the buildbot scripts will only run on 20.04 in fact.